.github/workflows/domain_ci.yml

```diff
@@ -145,3 +145,47 @@ jobs:
         run: pytest --no-header -v test/test_audio_examples.py
         env:
           PYTORCH_TEST_WITH_SLOW: 1
+
+  torcharrow:
+    if: ${{ github.repository_owner == 'pytorch' }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          # - macos-latest
+          - ubuntu-latest
+          # - windows-latest
+        python-version:
+          - 3.7
+          - 3.8
+          - 3.9
+    steps:
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install torch and torcharrow from nightlies
+        run: pip install --pre torch torcharrow -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```

Comment:
FWIW, when GitHub Actions provides `E5-2673 (v3 or v4) ` as CPU, today TorchArrow binary will encountered "Illegal Instruction" issue due to Velox requires some instructions not supported on that CPU :( 

https://github.com/facebookresearch/torcharrow/pull/92#issuecomment-982850395

Reply:
Hey @wenleix, I asked this question on the GIthub community forum. I don't think there's a way to do this yet. 
https://github.community/t/exclude-cpu-type-in-github-actions/243459

Reply:
We can add a step in this workflow to figure out the cpu model. If the model is among the exclusion list, we can skip all the following steps.

I only know we can use `lscpu` on linux. Not sure how to achieve it on windows machine though.

Reply:
@lezwon Can you make a change based on ejguan's suggestion above? Thanks!

Reply:
@NivekT sure thing.

Reply:
@NivekT @ejguan I'm not very sure how to go about adding a condition for the same in GitHub action. I have made some changes in the latest commit. Do let me know if its valid. :)

Reply:
cc: @wenleix It seems there are more CPU instruction sets not supported by Velox based on https://github.com/pytorch/data/runs/5989057961?check_suite_focus=true

Reply:
@ejguan I guess the "check if CPU is supported" doesn't really kick in?

```
Run if [ "$RUNNER_OS" == "Linux" ]; then
/home/runner/work/_temp/7b748253-1d41-4cd6-8083-de5[13](https://github.com/pytorch/data/runs/5989057961?check_suite_focus=true#step:2:13)e8058fd.sh: line 7: [: too many arguments
```

Reply:
Also maybe print cpuinfo so we can debug?

Reply:
Looks like it will not be necessary to check for certain CPUs anymore after this PR has landed:
https://github.com/facebookresearch/torcharrow/pull/315#issuecomment-1119151010

We can remove those related logic in this PR. Thanks @wenleix !